backend/app/minio/schemas.py

```
@@ -1,12 +1,16 @@
-
-
 from pydantic import BaseModel
 from typing import List
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)